assets/app/styles/_core.less

```diff
@@ -56,16 +56,9 @@ html, body {
 
 .console-os .navbar-project {
   border-bottom: 1px solid @navbar-os-project-border-color; 
-  padding-bottom: 5px;
+  padding-bottom: 20px;
   position: relative;
   background: @navbar-os-project-bg;
-  min-height: 85px;
```

Comment:
And if it ends up still being necessary, add a comment to explain what it's for so it doesn't get removed again :)